src/v/utils/to_string.h

```diff
@@ -50,6 +51,23 @@ operator<<(std::ostream& o, const ss::lowres_clock::duration& d) {
     return o;
 }
 
+template<typename Value, typename Hash, typename Pred, typename Allocator>
+std::ostream& operator<<(
+  std::ostream& os, const std::unordered_set<Value, Hash, Pred, Allocator>& v) {
+    bool first = true;
+    os << "{";
+    for (auto&& elem : v) {
+        if (!first) {
+            os << ", ";
+        } else {
+            first = false;
+        }
+        os << elem;
+    }
+    os << "}";
+    return os;
+}
```

Comment:
Issue: Allow defaulted additional template args
nitpick: prefer `fmt`

```suggestion
template<typename... Args>
std::ostream&
operator<<(std::ostream& os, const std::unordered_set<Args...>& v) {
    fmt::print(os, "{{{}}}", fmt::join(v, ", "));
    return os;
}
```
Technically you also need `<fmt/format.h>`

Reply:
Will also add the default template args for the stl container.